### PR TITLE
Add survey for CBBS imaging platform

### PR DESCRIPTION
**Pull request by bpoldrack**
Initial draft to add a survey for CBBS imaging platform.

Please review and leave your comments, @mih and @aqw !

Closes https://github.com/psychoinformatics-de/cbbs-imaging/issues/1

### CODE REVIEW

**Comment by mih** [comment]
Is it online?

File: www/cbbs_imaging/index.html [hunk 1]
```diff
@@ -0,0 +1,510 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0" />
+  <meta name="description" content="How-Do-You-Science Demo" />
```
Comment (aqw) [inline www/cbbs_imaging/index.html]: This should be adjusted.

File: www/cbbs_imaging/index.html [hunk 2]
```diff
@@ -0,0 +1,510 @@
...
+  <div class='timeline'>
+    <div class='tl-panel badge-university visible'>
+      <h2>Institution</h2>
+      <p>Select the institution you are from</p>
+      <select name='institution' required>
+          <option value='' disabled selected hidden>— Select —</option>
+          <option>OvGU (without Hospital)</option>
+          <option>Leibniz Institute for Neurobiology</option>
+          <option>University Hospital Magdeburg</option>
+          <option>German Centre for Neurodegenerative Diseases</option>
```
Comment (aqw) [inline www/cbbs_imaging/index.html]: Add "Other"?

File: www/cbbs_imaging/index.html [hunk 3]
```diff
@@ -0,0 +1,510 @@
...
+<form id='survey' autocomplete='off'>
+  <input name='client_version' type='hidden' value='1' />
+
+  <div class='timeline'>
+    <div class='tl-panel badge-university visible'>
+      <h2>Institution</h2>
+      <p>Select the institution you are from</p>
+      <select name='institution' required>
+          <option value='' disabled selected hidden>— Select —</option>
+          <option>OvGU (without Hospital)</option>
```
Comment (aqw) [inline www/cbbs_imaging/index.html]: s/without Hospital/University only/

File: www/cbbs_imaging/index.html [hunk 4]
```diff
@@ -0,0 +1,510 @@
...
+    </div>
+
+    <div class='tl-panel badge-university'>
+      <h2>Lab</h2>
+      <p>What lab are you from?</p>
+      <input name='lab' type='text' required>
+    </div>
+
+    <div class='tl-panel badge-university'>
+      <h2>How many people are doing data analyses in your lab?</h2>
```
Comment (aqw) [inline www/cbbs_imaging/index.html]: Are we emphasizing data here? It just feels odd. Hard to analyze without data.

File: www/cbbs_imaging/index.html [hunk 5]
```diff
@@ -0,0 +1,510 @@
...
+        <option>Other</option>
+        <!-- TODO: Other => text input?
+             TODO: Linux => more specific if choosen?
+        -->
+      </select>
+    </div>
+
+    <div class='tl-panel inverted badge-laptop'>
+      <h2>Shared infrastructure</h2>
+      <p>Does your lab <em>use</em> shared computing infrastructure?</p>
```
Comment (aqw) [inline www/cbbs_imaging/index.html]: "such as" might help

File: www/cbbs_imaging/index.html [hunk 6]
```diff
@@ -0,0 +1,510 @@
...
+
+    <div class="tl-panel badge-heart">
+      <h2>Desired analyses</h2>
+      <p>What analysis procedures/workflows would you most appreciate us to make available?</p>
+      <input name="analyses_desired" type="text">
+    </div>
+
+    <div class="tl-panel badge-upload">
+      <h2>Your contribution</h2>
+      <p>What methods/tools developed in your lab would you like to become available?</p>
```
Comment (aqw) [inline www/cbbs_imaging/index.html]: For others, for self? What's primary motivation? I think people will parse this statement.

File: www/cbbs_imaging/index.html [hunk 7]
```diff
@@ -0,0 +1,510 @@
...
+        <label><input type='checkbox' name='vcs_data_list[]'
+          value='Subversion'> Subversion</label>
+        <label><input type='checkbox' name='vcs_data_list[]'
+          value='git-annex'> git-annex</label>
+        <label><input type='checkbox' name='vcs_data_list[]'
+          value='git-lfs'> git-lfs</label>
+        <label><input type='checkbox' name='vcs_data_list[]'
+          value='ipfs'> ipfs</label>
+        <label><input type='checkbox' name='vcs_data_list[]'
+          value='dat'> dat</label>
```
Comment (aqw) [inline www/cbbs_imaging/index.html]: datalad?

File: www/cbbs_imaging/index.html [hunk 8]
```diff
@@ -0,0 +1,510 @@
...
+      <select name="used_bids" required>
+        <option value="" disabled selected hidden>— Select —</option>
+        <option>Yes</option>
+        <option>No</option>
+      </select>
+    </div>
+
+    <div class="tl-panel badge-search">
+      <h2>Scanners</h2>
+      <p>What Scanners are used by your lab?</p>
```
Comment (aqw) [inline www/cbbs_imaging/index.html]: Scanners should be lower case

File: www/cbbs_imaging/index.html [hunk 9]
```diff
@@ -0,0 +1,510 @@
...
+          value='prisma'> 3T Siemens Prisma</label>
+        <label><input type='checkbox' name='scanners[]'
+          value='achieva'> 3T Philips Achieva</label>
+        <label><input type='checkbox' name='scanners[]'
+          value='meg'> MEG</label>
+      </div>
+    </div>
+
+    <div class="tl-panel badge-database">
+      <h2>Amount of acquisitions</h2>
```
Comment (aqw) [inline www/cbbs_imaging/index.html]: Amount is wrong word. TODO: reword (uncertain of my suggestion)

File: www/cbbs_imaging/index.html [hunk 10]
```diff
@@ -0,0 +1,510 @@
...
+      <select name='custom_logs' required>
+        <option value='' disabled selected hidden>— Select —</option>
+        <option>Standard</option>
+        <option>Custom</option>
+      </select>
+    </div>
+
+    <div class="tl-panel badge-clock">
+      <h2>Time until publication</h2>
+      <p>What's the timeframe from end of acquisition until submission of a publication?</p>
```
Comment (aqw) [inline www/cbbs_imaging/index.html]: The "typical" timeframe

File: www/cbbs_imaging/success.html [hunk 11]
```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <meta name="description" content="How-do-you-science survey" />
+  <meta name="author" content="Psychoinformatics Lab" />
+
+  <title>Victory! — How Do You Science?</title>
```
Comment (aqw) [inline www/cbbs_imaging/success.html]: Update

File: www/cbbs_imaging/success.html [hunk 12]
```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <meta name="description" content="How-do-you-science survey" />
```
Comment (aqw) [inline www/cbbs_imaging/success.html]: Update

**Review by aqw — CHANGES_REQUESTED** [review]

**Comment by bpoldrack** [comment]
Thanks, @aqw.

Online now: http://howdoyouscience.net/cbbs_imaging/

**Comment by mih** [comment]
Merging, final touches in master

**Pull request merged**